### PR TITLE
Variation docs update - e102

### PR DESCRIPTION
**Pull request by ima23**
sandbox: http://ves-hx2-76.ebi.ac.uk:6080/

### CODE REVIEW

File: ensembl/htdocs/info/genome/variation/species/sources_documentation.html [hunk 1]
```diff
@@ -83,15 +83,15 @@
...
+      <span class="_ht vdoc_source_count vdoc_new_version vdoc_new_version_count" title="2 new data versions">2</span>
     </div>
     <div style="clear:both"></div>  
   </div>
   
   <div style="margin:0px 4px 5px">
     <div class="vdoc_menu_left">
       <img src="/i/species/Canis_lupus_familiaris.png" alt="Dog" class="badge-16" style="margin-right:4px;vertical-align:middle" />
-      <a href="#canis_lupus_familiaris" style="margin-right:3px;text-decoration:none;vertical-align:middle" title="Canis lupus familiaris">Dog</a>
+      <a href="#canis_lupus_familiaris" style="margin-right:3px;text-decoration:none;vertical-align:middle" title="Canis lupus_familiaris">Dog</a>
```
Comment (aparton) [inline ensembl/htdocs/info/genome/variation/species/sources_documentation.html:94]: any idea why an underscore has appeared in the title here?

File: ensembl/htdocs/info/genome/variation/species/sources_documentation.html [hunk 2]
```diff
@@ -1927,10 +1936,10 @@ <h2 id="bos_taurus" style="display:inline;color:#333">Cow<span class="small vdoc
   
     </tr>
     </table>
-  </div><div style="height:20px"></div><!-- Dog (Dog) -->
+  </div><div style="height:20px"></div><!-- Dog (Canis lupus_familiaris) -->
     <div style="padding-left:0px;padding-bottom:3px">
-      <a href="/canis_lupus_familiaris/Info/Index" title="Dog Ensembl Home page" style="vertical-align:middle" target="_blank"><img src="/i/species/Canis_lupus_familiaris.png" alt="Dog" class="badge-48" style="float:none;margin-right:4px;vertical-align:middle" /></a>
-      <h2 id="canis_lupus_familiaris" style="display:inline;color:#333">Dog<span class="small vdoc_species_sci_name"> (Canis lupus familiaris)</span></h2>
+      <a href="/Canis_lupus_familiaris/Info/Index" title="Dog Ensembl Home page" style="vertical-align:middle" target="_blank"><img src="/i/species/Canis_lupus_familiaris.png" alt="Dog" class="badge-48" style="float:none;margin-right:4px;vertical-align:middle" /></a>
+      <h2 id="canis_lupus_familiaris" style="display:inline;color:#333">Dog<span class="small vdoc_species_sci_name"> (Canis lupus_familiaris)</span></h2>
```
Comment (aparton) [inline ensembl/htdocs/info/genome/variation/species/sources_documentation.html:1942]: Underscore appears in dog name again

File: ensembl/htdocs/info/genome/variation/species/sources_documentation.html [hunk 3]
```diff
@@ -3164,6 +3173,180 @@ <h2 id="saccharomyces_cerevisiae" style="display:inline;color:#333">S. cerevisia
   
       </tr>
     </table>
+  </div><div style="height:20px"></div><!-- Sheep (Ovis aries_rambouillet) -->
+    <div style="padding-left:0px;padding-bottom:1px">
+      <a href="/Ovis_aries_rambouillet/Info/Index" title="Sheep Ensembl Home page" style="vertical-align:middle" target="_blank"><img src="/i/species/Ovis_aries_rambouillet.png" alt="Sheep" class="badge-48" style="float:none;margin-right:4px;padding:2px;vertical-align:middle;background-color:#00F" /></a>
+      <h2 id="ovis_aries_rambouillet" style="display:inline;color:#333">Sheep<span class="small vdoc_species_sci_name"> (Ovis aries_rambouillet)</span></h2><span style="padding-left:20px;color:#00F;font-weight:bold">New species!</span>
```
Comment (aparton) [inline ensembl/htdocs/info/genome/variation/species/sources_documentation.html:3179]: underscore in sheep name

File: docs/htdocs/info/docs/tools/vep/script/vep_download.html [hunk 4]
```diff
@@ -116,14 +116,19 @@ <h2 id="new">What's new?</h2>
     <h4>New in version [[SPECIESDEFS::ENSEMBL_VERSION]] <i>([[SPECIESDEFS::ENSEMBL_RELEASE_DATE]])</i></h4>
 
     <ul>
-      <li>New options:
+      <li>VEP options:
         <ul>
-          <li><a href="vep_options.html#opt_var_synonyms">--var_synonyms</a>: Report known synonyms for colocated variants. Must be used with <a href="vep_options.html#opt_cache">--cache</a>.</li>
+          <li><a href="vep_options.html#opt_uniprot">--uniprot</a>: Now we report precise Ensembl translation to UniProt isoform mappings.</li>
```
Comment (aparton) [inline docs/htdocs/info/docs/tools/vep/script/vep_download.html:121]: We could also add:
--spdi - https://github.com/Ensembl/ensembl-vep/pull/828
Variant Recoder allele specific changes - https://github.com/Ensembl/ensembl-vep/pull/839

File: docs/htdocs/info/docs/tools/vep/script/vep_filter.html [hunk 5]
```diff
@@ -97,6 +97,14 @@ <h2 id="filter_opt">Options</h2>
                 line to be printed
 			</td>
 		</tr>
+    <tr id="opt_softfilter">
+      <td><pre>--soft_filter</pre></td>
+      <td></td>
+      <td>
+         Variation failing the filter will not get excluded from the output file,
```
Comment (aparton) [inline docs/htdocs/info/docs/tools/vep/script/vep_filter.html:104]: I suggest changing to:

`Variants not passing given filters will be flagged in the FILTER column of the VCF file, and will not be removed from output.`

File: docs/htdocs/info/docs/api/variation/variation_schema.html [hunk 6]
```diff
@@ -1815,13 +1811,13 @@ <h2>Other tables</h2>
         <li class="sql_schema_species_name">Equus caballus</li>
         <li class="sql_schema_species_name">Felis catus</li>
         <li class="sql_schema_species_name">Gallus gallus</li>
-        <li class="sql_schema_species_name">Homo sapiens</li>
```
[the remark this answers is not in the document]
Comment (ima23) [reply docs/htdocs/info/docs/api/variation/variation_schema.html:1818]: It's now removed as human does not have data in that table anymore.

File: docs/htdocs/info/docs/tools/vep/script/vep_download.html (hunk 4 above)
Comment (helensch) [reply docs/htdocs/info/docs/tools/vep/script/vep_download.html:121]: vep#839 is for the next release

**Review by helensch — APPROVED** [review]
Looks fine with suggested updates

File: docs/htdocs/info/docs/tools/vep/script/vep_download.html (hunk 4 above)
Comment (ima23) [reply docs/htdocs/info/docs/tools/vep/script/vep_download.html:121]: have added `--spdi` option added in the latest commit

**Comment by ima23** [comment]
Thanks for review @aparton and @helensch, PR is ready for final check. Thanks.

**Comment by imransl** [comment]
@ima23 @helensch There are some conflicts to be resolved before merging. I can merge the PR once it's done.

File: docs/htdocs/info/docs/tools/vep/script/vep_options.html [hunk 7]
```diff
@@ -1701,7 +1713,7 @@ <h2 id="filt">Filtering and QC options</h2>
         <p>Customise the order of criteria (and the list of criteria) applied when choosing a block of
         annotation data with one of the following options: <a href="#opt_pick">--pick</a>, <a href="#opt_pick_allele">--pick_allele</a>, <a href="#opt_per_gene">--per_gene</a>, <a href="#opt_pick_allele_gene">--pick_allele_gene</a>, <a href="#opt_flag_pick">--flag_pick</a>, <a href="#opt_flag_pick_allele">--flag_pick_allele</a>, <a href="#opt_flag_pick_allele_gene">--flag_pick_allele_gene</a>. See
         <a href="vep_other.html#pick_options">this page</a> for the default order.<br />
-        Valid criteria are: [ <i>canonical appris tsl biotype ccds rank length mane</i> ]. e.g.:</p>
+        Valid criteria are: [ <i>canonical appris tsl biotype ccds rank length</i> ]. e.g.:</p>
```
Comment (aparton) [inline docs/htdocs/info/docs/tools/vep/script/vep_options.html:1716]: mane is still a valid option for pick

**Review by aparton — APPROVED** [review]

**Pull request merged**